optimize/backend/src/main/java/io/camunda/optimize/rest/engine/EngineContext.java

```diff
@@ -100,7 +86,7 @@ public Client getEngineClient() {
   }
 
   public void close() {
```

Comment:
not for this PR clearly, but do you have an expectation/issue for when this class can be deleted completely? Or rather is it your expectation that this happens slowly over the course of the issues following this one?

Reply:
I think we should be able to remove this once the status checking and engine authentication is removed. Status checking is easy and coming up next, engine auth logic is a bit chunkier

Reply:
Nice!